<launch>
    <arg name="world_T_base_topic" default="/mur620/mir_pose_stamped_simple"/>
    <arg name="twist_out" default="/ur_twist_world_in_mir"/>
    <arg name="twist_in" default="/ur_twist_world"/>
    <node name="world_twist_in_mir" pkg="ur_trajectory_follower" type="rotate_twist_by_pose.py" output="screen">
        <remap from="twist_in" to="$(arg twist_in)"/>
        <remap from="base_twist" to="$(arg twist_out)"/>
        <remap from="world_T_base" to="$(arg world_T_base_topic)"/>
    </node>
</launch>